test/extensions/funding-queue.js

```diff
@@ -558,7 +558,7 @@ contract("Funding Queues", (accounts) => {
       // So 1 - (1 - 1/2 * 2/3) = 1/3 (33.3%) of the balance should be transferred
       const balanceAfter = await colony.getFundingPotBalance(1, token.address);
       const amountTransferred = balanceBefore.sub(balanceAfter);
-      const expectedTransferred = new BN("333743300899454444");
+      const expectedTransferred = new BN("333743300899535035");
```

Comment:
I'm curious why the differences here are so larger? (Larger than 1 wei I mean).

Reply:
The values that are calculated using `wmul`/`wdiv` are [raised to a large power](https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/extensions/FundingQueue.sol#L360) in the calculation for how much funding to transfer, and so a 1 wei difference in those numbers is going to result in a large discrepancy here.